packages/cursorless-engine/src/actions/PasteFromClipboard.ts

```diff
@@ -29,32 +26,38 @@ export class PasteFromClipboard {
     // First call editNew in order to insert delimiters if necessary and leave
     // the cursor in the right position. Note that this action will focus the
     // editor containing the targets
-    const [originalCursorSelections] = await callFunctionAndUpdateSelections(
-      this.rangeUpdater,
-      async () => {
-        await this.actions.editNew.run(destinations);
-      },
-      editor.document,
-      [editor.selections],
-    );
+    const callbackEdit = async () => {
+      await this.actions.editNew.run(destinations);
+    };
```

Comment:
i def seem to prefer inlining a bit more than you do, but I don't think it matters much 😊. Fwiw I think we could simplify this one by removing the `async` and `await`, in which case it is pretty strong candidate for inlining

Reply:
Unfortunately we can't do that since they have different return values. editNew returns an action results, but our callback expects void.